src/main/java/teammates/ui/webapi/output/CommonSearchLinksData.java

```diff
@@ -0,0 +1,52 @@
+package teammates.ui.webapi.output;
+
+/**
+ * Contains the common links for instructors and students.
+ */
+public class CommonSearchLinksData {
+    protected boolean showLinks;
+    protected String email;
```

Comment:
I see that you used `@Nullable` for nullable strings in other classes. Maybe these fields should be `nullable` too to be consistent?

Reply:
The `@Nullable` annotation is used in other classes to provide hints for the TypeScript generator so that it can generate optional types. In this case, the types are not optional so I don't think the annotation should be added here.

Reply:
Oh as in these fields are set to `null` in the constructor.. Doesn't that imply that they are optional?